build.xml

```diff
@@ -86,6 +86,8 @@
         <include name="compiler/builtins-serializer/src"/>
         <include name="compiler/cli/src"/>
         <include name="compiler/cli/cli-common/src"/>
+        <include name="compiler/rmi/rmi-server/src"/>
+        <include name="compiler/rmi/rmi-interface/src"/>
```

Comment:
There's another fileset which is used in the `compiler-quick` target and declared just below, please also add these modules there.

Also please add them to `compiler-sources`.